### DISCUSSION TITLE
Set Application Name

### DISCUSSION THREAD

**Discussion by Tigerlady** [MasaCMS/MasaCMS#260] [Q&A]
Is there a way to set a specific Application name instead of using the default MASA and a bunch of numbers?  I am using Fusion Reactor, and all the MASA apps show up as MASA and numbers.  I want to distinguish which applications belong to which websites.

**Comment by no3rdw** [comment]
It looks like this can be overridden by adding a line to the [settings] section of config/settings.ini.cfm:
appname=yourcustomname

I haven't actually tried this, so do so at your own risk :D
